library/core/src/ffi/c_str.rs

```diff
@@ -79,8 +79,8 @@ use crate::{fmt, ops, slice, str};
 ///
 /// fn my_string_safe() -> String {
 ///     let cstr = unsafe { CStr::from_ptr(my_string()) };
-///     // Get copy-on-write Cow<'_, str>, then guarantee a freshly-owned String allocation
-///     String::from_utf8_lossy(cstr.to_bytes()).to_string()
+///     // Get a copy-on-write Cow<'_, str>, then guarantee a freshly-owned String allocation
+///     cstr.to_string_lossy().to_string()
```

Comment:
`into_owned` avoid reallocation, but `to_string()` doesn't.
```suggestion
///     cstr.to_string_lossy().into_owned()
```